src/sourcemaps/sourcemaps-wizard.ts

```diff
@@ -220,7 +229,11 @@ async function startToolSetupFlow(
     case 'angular':
       await configureSentryCLI(
         options,
-        configureAngularSourcemapGenerationFlow,
+        // Angular wizard handles the angular.json setup itself
+        !preSelectedTool || preSelectedTool !== 'angular'
+          ? configureAngularSourcemapGenerationFlow
+          : // Not leaving this as undefined to avoid default. This is expected to be a no-op.
+            async () => Promise.resolve(),
```

Comment:
I see there's a new paramter (`skipValidation`). Should we use it here instead of creating the noop function?